src/sentry/replays/endpoints/organization_issue_replay_count.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from typing import Any
+
+from rest_framework import status
+from rest_framework.response import Response
+from snuba_sdk import Column, Condition, Op, Request
+
+from sentry import features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import NoProjects
+from sentry.api.bases.organization_events import OrganizationEventsV2EndpointBase
+from sentry.models import Organization
+from sentry.replays.query import query_replays_dataset
+from sentry.search.events.builder import QueryBuilder
+from sentry.utils.snuba import Dataset, raw_snql_query
+
+
+@region_silo_endpoint
+class OrganizationIssueReplayCountEndpoint(OrganizationEventsV2EndpointBase):
+    """
+    Get all the replay ids associated with a set of issues in discover,
+    then verify that they exist in the replays dataset, and return the count.
+    """
+
+    private = True
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        if not features.has("organizations:session-replay", organization, actor=request.user):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization, check_global_views=False)
```

Comment:
nit: but do you mind trying get_snuba_dataclass? Might make your life easier if you're typing the endpoint too

Reply:
why do I have to pass in the old params too if I'm using the dataclass? seems weird, but I'll update.

Reply:
🤔 you shouldn't need the old dict-style params anymore if you use this method